percona_tests/metrics_test.go

```diff
@@ -326,6 +326,10 @@ func testForMissingMetricsLabels(oldMetricsCollection, newMetricsCollection Metr
 func testForMissingMetrics(oldMetricsCollection, newMetricsCollection MetricsCollection) (bool, string) {
 	missingMetrics := make([]string, 0)
 	for metricName := range oldMetricsCollection.LabelsByMetric {
+		if metricName == "go_memstats_gc_cpu_fraction" {
```

Comment:
can you please document (add comment?) why it is skipped

Reply:
yep, const and explanation